spec/models/enterprise_caching_spec.rb

```diff
@@ -5,7 +5,8 @@
 describe Enterprise do
   context "key-based caching invalidation" do
     describe "is touched when a(n)" do
-      let(:enterprise) { create(:distributor_enterprise, updated_at: Time.zone.now - 1.week) }
+      # Reloading changes timestamps to the precision of the database.
+      let(:enterprise) { create(:distributor_enterprise, updated_at: 1.week.ago).reload }
```

Comment:
Do we need to reload the enterprise upon creation, and before anything action touched it?